tests/test_rest_api.py

```diff
@@ -348,7 +348,7 @@ def test_cache(airtemp_ds):
 
     response1 = client.get('/zarr/air/0.0.0')
     assert response1.status_code == 200
-    assert '/air/0.0.0' in rest.cache
+    assert 'airtemp/air/0.0.0' in rest.cache
```

Comment:
All datasets now have an `_xpublish_id` attr so the cache string changed in this test